index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src="https://kit.fontawesome.com/ea6955352c.js" crossorigin="anonymous"></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#projects>Protfoliu</a></li>
+            <li><a class="mobile-link" href="#about-me">About</a></li>
+            <li>
+              <a class="mobile-link" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <nav id="desktop-nav>
```

Comment:
The same error appears here, and the error is because of an unclosing quote, kindly fix it by closing the quote to make the project like the Figma design.

<img width="375" alt="Screenshot 2023-04-25 at 08 42 42" src="https://user-images.githubusercontent.com/91237525/234279643-2346a73d-3b98-4f8b-a212-a1dd7613c686.png">